Sources/Target/ProcessBase.cpp

```diff
@@ -202,6 +203,7 @@ ErrorCode ProcessBase::afterResume() {
   // Disable breakpoints and try to hit the breakpoint.
   //
   disableBreakpointManager(softwareBreakpointManager());
+  disableBreakpointManager(hardwareBreakpointManager());
```

Comment:
I see that the next commit adds these, and I understand what you're trying to achieve. Are people gonna hate me if I propose something like this:
```
for (auto e : { softwareBreakpointManager(), hardwareBreakpointManager() }) {
    if (e != nullptr)
        e->enable();
}
````

I like to avoid adding tiny functions like these if I can. While they do avoid some level of code duplication, they clutter the header file with useless APIs and C++ being very verbose you have to pass information around needlessly.